uninstall.php

```diff
@@ -3,20 +3,20 @@
  * @package Polylang
  */
 
-if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) { // If uninstall not called from WordPress exit
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) { // If uninstall not called from WordPress exit.
 	exit;
 }
 
 /**
- * Manages Polylang uninstallation
- * The goal is to remove ALL Polylang related data in db
+ * Manages Polylang uninstallation.
+ * The goal is to remove ALL Polylang related data in db.
```

Comment:
```suggestion
 * The goal is to remove **all** Polylang related data in db.
```
Let's be **bold** 😎
Maybe one day the doc will be parse into HTML of some sort 😅